.github/docker/Dockerfile.gorgone-testing-jammy

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:jammy
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Set locale
+RUN apt-get update && \
+    apt-get install -y locales libcurl4-openssl-dev curl wget zstd jq lsb-release mariadb-client iproute2 && \
+    apt-get install -y ca-certificates apt-transport-https software-properties-common gnupg2 && \
+    localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+
+ENV LANG=en_US.UTF-8
+
+# Add Centreon repositories and their public key
+RUN echo "deb https://packages.centreon.com/ubuntu-standard-24.05-testing/ jammy main" | tee -a /etc/apt/sources.list.d/centreon-testing.list && \
```

Comment:
isn't there a way to variabilize our current cloud version there instead of using "24.05"? i feel like that's something we can easily forget when changing versions